### ISSUE TITLE
Child Entity Producing Blocks Can't Cange State After An Initial Right Click

### ISSUE THREAD

**Issue by Sullos** [tsunamayo/Skywanderers#571]
I'll try and put it in simpler terms, take, for example, a hinge a rotor, or a slider, and if you right click it, we have it move the slider to one end of the slider rail, and if you right click it again, it goes to the other end. 
After the most recent update of bug fixes, it seems that trying to right click a slider or any other child entity creating block, it moves to one position, but then right clicking again doesn't move it back to its other position. 
![20180304092432_1](https://user-images.githubusercontent.com/33239800/36940020-67f8b11e-1f8f-11e8-9bf7-f7f15b85c540.jpg)

As in the above picture, we can see the sliders have moved their slider panel to one end of the orange slider block. Trying to right-click it again does nothing. 

I have been able to recreate this issue with the 3 hinges, the rotators, the rotor, and the two sizes of sliders. 
It's as if the hand when right clicking is only able to send one logic signal, and not the other one. As in we cant toggle them with our hand anymore.

If my bug report doesn't make any sense feel free to ask for a new report that hopefully makes more sense.

*Edit*
After a bit more playing recently, the issue seems to be the same for small brick hinges, and mirror blocks too.
The way most have described it is to say the player's hand is outputting a value of 1 when right clicking and not toggling by sending values of 1 and 0.
I'll add more info if I find anything else.

**Comment by tsunamayo** [comment]
Thanks, this issue was finally fixed. Cheers
